src/main/resources/application.properties

```diff
@@ -1,38 +1,40 @@
 server.port=8090
 
 #Environment Set-up configurations
-org.ops4_0.constants.port=
+org.ops4_0.constants.port=8090
 org.ops4_0.constants.host=http://localhost:
-org.ops4_0.constants.schedulerHostUrl=
+org.ops4_0.constants.schedulerHostUrl=https://jarvis4ops.herokuapp.com/
 
 #Slack Constants
 org.ops4_0.constants.greatJobTitleMsg=did a great job !! HURRAYYYYY!!!
 org.ops4_0.constants.incidentsResolvedMsg=Resolved 5+ incidents yesterday, congratulations for the good effort.
 org.ops4_0.constants.cfd=#ClientFocusedDelivery
 org.ops4_0.constants.teamWork=#TeamWork
-org.ops4_0.constants.incidentRockstarMemeList=https://cdn.meme.am/cache/instances/folder636/200x/74587636.jpg,https://cdn.meme.am/cache/instances/folder529/200x/74616529.jpg,https://cdn.meme.am/cache/instances/folder443/200x/74630443.jpg,https://cdn.meme.am/cache/instances/folder374/200x/74631374.jpg,https://cdn.meme.am/cache/instances/folder67/200x/74631067.jpg
+org.ops4_0.constants.invalidTimeMessage=Following incidents are closed in an invalid time
+org.ops4_0.constants.incidentRockstarMemeList=https://cdn.meme.am/cache/instances/folder636/500x/74587636.jpg,https://cdn.meme.am/cache/instances/folder529/500x/74616529.jpg,https://cdn.meme.am/cache/instances/folder443/500x/74630443.jpg,https://cdn.meme.am/cache/instances/folder374/500x/74631374.jpg,https://cdn.meme.am/cache/instances/folder67/500x/74631067.jpg
 
 #JIRA Details
 org.ops4_0.constants.jiraCreds=
-org.ops4_0.constants.jiraEndPoint=
-org.ops4_0.constants.sampleJiraEndPoint=
+org.ops4_0.constants.jiraEndPoint=https://jira.devops.mnscorp.net/rest/api/2/search?
```

Comment:
Don't check-in values please. Remove them - for all below properties too.
Don't publish the jira portal path too.